forest/config.py

```diff
@@ -84,14 +101,21 @@ def load(cls, path):
                   file_type: rdt
 
         :param path: JSON/YAML file to load
+        :param variables: dict of key/value pairs used by :py:`string.Template`
         :returns: instance of :class:`Config`
         """
         with open(path) as stream:
-            try:
-                # PyYaml 5.1 onwards
-                data = yaml.full_load(stream)
-            except AttributeError:
-                data = yaml.load(stream)
+            text = stream.read()
+
+        if variables is not None:
+            template = string.Template(text)
+            text = template.substitute(**variables)
+
+        try:
+            # PyYaml 5.1 onwards
+            data = yaml.safe_load(text)
+        except AttributeError:
+            data = yaml.load(text)
```

Comment:
It'd be safer just to mandate a sufficiently recent version of PyYAML and ditch the fallback.

Reply:
I think the Docker container running on EC2 is due an update if that's the approach we take, possibly a separate PR?